packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -896,3 +984,86 @@ class _Transform {
     this.batch,
   );
 }
+
+/// A mutable version of [ui.RSTransform] for custom batch manipulation.
+class _RSTransform implements ui.RSTransform, PositionProvider {
+  final _values = Float32List(4);
+
+  /// This is a cache of `-scos * anchorX + ssin * anchorY`
+  final double _anchorX;
+
+  /// This is a cache of `-ssin * anchorX - scos * anchorY`
+  final double _anchorY;
```

Comment:
Couldn't these quite easily be replaced by `AnchorProvider`?

Reply:
TL/DR: I don't think we should.

`Anchor` is relative (0.0 to 1.0) - whereas `RSTransform` anchor is a coordinate in the Atlas image (could be a single source image, could be a giant atlas - which now that I look at Orthogonal, it's probably wrong for rotation!). 

`Anchor`:
```dart
  /// The relative x position with respect to the object's width;
  /// 0 means totally to the left (beginning) and 1 means totally to the
  /// right (end).
```

`RSTransform` calculations that are added:

```
The x coordinate of the translation, minus [scos](https://api.flutter.dev/flutter/dart-ui/RSTransform/scos.html) multiplied by the x-coordinate of the rotation point, plus [ssin](https://api.flutter.dev/flutter/dart-ui/RSTransform/ssin.html) multiplied by the y-coordinate of the rotation point.
```


Right now the anchor is hardcoded to the source middle: 

```dart
        final anchorX = src.width / 2; // I think this is wrong for "atlas" with more than 1 image.
// or
        final anchorX = src.width - halfMapTile.x; // I think this is correct.
```

Lastly: what would they ultimately animate and modify? Let's assume we cache the src rectangle and convert anchor to relative - then yes, you could add an [AnchorEffect](https://docs.flame-engine.org/1.3.0/flame/effects.html#anchorbyeffect), but I guess I'm not seeing the usefulness of that with Tiles.

I am happy to be wrong. If you want me to dig into it, I think we could do that as another PR to add more functionality. Thoughts?